source/features/extend-conversation-status-filters.tsx

```diff
@@ -34,7 +34,7 @@ function addMergeLink(): void {
 		const mergeLink = lastLink.cloneNode(true);
 		mergeLink.textContent = 'Merged';
 		mergeLink.classList.toggle('selected', new SearchQuery(location.search).includes('is:merged'));
-		new SearchQuery(mergeLink).replace('is:closed', 'is:merged');
+		new SearchQuery(mergeLink).replace('is:closed', 'is:merged').applyChanges();
```

Comment:
Thank you! I was thinking about dropping the internal `link` altogether:


```suggestion
		mergeLink.href = SearchQuery.from(mergeLink)
			.replace('is:closed', 'is:merged')
			.href;
```

Internally:

`SearchQuery.from` is `() => return new SearchQuery(link.href)`

Reply:
> I was thinking about dropping the internal `link` altogether

:+1:

> `SearchQuery.from` is `() => return new SearchQuery(link.href)`

We'd still need to pass the link element, or at least its `.pathname`, since some methods need to take a look at it.

Reply:
`link.href` uses the whole URL